opennms-provision/opennms-provisiond/src/main/java/org/opennms/netmgt/provision/service/snmp/IpAddrTable.java

```diff
@@ -227,8 +229,7 @@ public void updateIpInterfaceData(OnmsNode node, String ipAddr) {
             ipIf.setSnmpInterface(snmpIf);
 
         }
-
-        ipIf.setIpHostName(ipAddr);
```

Comment:
This is one of the place where existing hostname in ipInterface is getting overwritten